packages/core/core/src/assetUtils.js

```diff
@@ -156,14 +156,18 @@ export async function getConfig(
     packageKey?: string,
     parse?: boolean,
   |},
-): Promise<ConfigResult | null> {
+): Promise<ConfigOutput | null> {
   let packageKey = options?.packageKey;
   let parse = options && options.parse;
 
   if (packageKey != null) {
     let pkg = await asset.getPackage();
     if (pkg && pkg[packageKey]) {
-      return pkg[packageKey];
+      return {
+        config: pkg[packageKey],
+        // The package.json file was already registered by asset.getPackage() -> asset.getConfig()
+        files: [],
```

Comment:
Should `files` be "source files for the config result" or "files that will need to be added as included files"?
For the former, we'd have to return the package.json path (`getPackage()` returns only the result) as well.